build/yamls/flow-aggregator.yml

```diff
@@ -203,6 +203,21 @@ data:
 
       # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
       #tlsMinVersion:
+    
+    # clickHouse contains ClickHouse related configuration options.
+    #clickHouse:
+      # Username of ClickHouse client.
+      #username: "clickhouse_operator"
+      # Password of ClickHouse client.
+      #password: "clickhouse_operator_password"
```

Comment:
Do we need to specify clickhous creds twice, or can we infer those from the relevant secret in flow aggregator?

Reply:
Question - I do not see cred is specified twice in this YAML. Is it needed anywhere else too?

Reply:
@jianjuns it's not in this PR. The flow visibility has the same cred under: https://github.com/antrea-io/antrea/blob/9c4b86c67d3f6c2b42d185b314f9d99e3aba78b0/build/yamls/flow-visibility.yaml#L4641

Reply:
I think we can ignore this comment for the time being.